.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+---
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
```

Comment:
`actions/checkout` action fetches only last commit, but `helm/chart-releaser-action` uses `git diff` to determinate what HELM chart was changed and should be released.

So we have to add `fetch-depth: 0` param to fetch al git history, like in example here:
https://github.com/helm/chart-releaser-action#user-content-example-workflow

```
      - name: Checkout
        uses: actions/checkout@v2
        with:
          fetch-depth: 0
```

Reply:
Done!